packages/sync-actions/src/shipping-methods.js

```diff
@@ -40,6 +41,11 @@ function createShippingMethodsMapActions(
         )
       )
     )
+    allActions.push(
+      mapActionGroup('custom', (): Array<UpdateAction> =>
+        actionsMapCustom(diff, oldObj, newObj)
```

Comment:
`actionsMapCustom` is tricky 🙄 cause the order is `(diff, newObj, oldObj)` unlike the other actions
https://github.com/commercetools/nodejs/blob/96e131866ec4aeaf40494790f5292e733fc7645e/packages/sync-actions/src/categories.js#L48-L52

already attached unit test, so please feel free to confirm the switch 
```suggestion
        actionsMapCustom(diff, newObj, oldObj)
```